release.json

```diff
@@ -8,7 +8,7 @@
     "nightly": {
         "INTEGRATIONS_CORE_VERSION": "master",
         "OMNIBUS_SOFTWARE_VERSION": "8664357efd9187e1c16284e3d9811ea25d152186",
-        "OMNIBUS_RUBY_VERSION": "78573b961ef3400787a7069526d91795c1a34238",
+        "OMNIBUS_RUBY_VERSION": "chouquette/tmp_split_build_packaging",
```

Comment:
Did you forget to remove your test ?

Reply:
No, currently omnibus doesn't support some of the things we need, so I need to keep this for the builds to succeed until the changes have been merged there (https://github.com/DataDog/omnibus-ruby/pull/200 & https://github.com/DataDog/omnibus-ruby/pull/201) but indeed these changes will need to be removed eventually